### DISCUSSION TITLE
Github student authentication fails due to an error in Github. What should I do?

### DISCUSSION THREAD

**Discussion by suku1515** [community/community#58891] [GitHub Education]
### Hi everyone,

I am unable to attempt student authentication right now because of an error in Github. The reason is that if you look at the picture, it should have been MokpoYudal Middle School, but it is forcibly written as Mokpo High School. I tried to change the name of this school because it was not listed correctly, so I made it impossible to change it. So, no matter how much I tried with my student ID and certificate of enrollment, I couldn't do it because of the same "school name is not listed". I tried to look for this problem, but I couldn't find that the name of the school was different in ah, so I couldn't try it. What should I do? help!
![제발](https://github.com/community/community/assets/136946245/1019902c-fb8a-461d-98b9-1c44faa9ba2e)

**Comment by samus-aran** [comment]
Hi @suku1515 ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the [same comment in multiple places](https://github.com/orgs/community/discussions/58825). As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
